index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Ola Ishola | Software Developer</title>
+    <script
+      src="https://kit.fontawesome.com/792c795fd6.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="wrapper">
+      <nav class="navbar">
+        <ul>
+          <li class="logo"><a href="#">Olaish</a></li>
+          <li class="menu"><i class="fas fa-bars"></i></li>
+        </ul>
+      </nav>
+      <div class="hero">
+        <section class="headline">
+          <div class="title">
+            <h2>I’m Ola Glad to see you!</h2>
+          </div>
+
+          <div class="description">
+            <p>
+              I’m a software developer! I can help you build a product , feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+          </div>
+
+          <div class="social">
+            <p>LET’S CONNECT</p>
```

Comment:
- The button is a link ``<a>`` or ``<button>`` tag, and follow the button styling: Active, Hover and Pressed.